frontend/viewer/src/project/browse/EntryRow.svelte

```diff
@@ -5,17 +5,20 @@
   import type {Snippet} from 'svelte';
   import {t} from 'svelte-i18n-lingui';
   import DictionaryEntry from '$lib/DictionaryEntry.svelte';
-  import ListItem from '$lib/components/ListItem.svelte';
-  import {cn} from '$lib/utils';
+  import ListItem, {type ListItemProps} from '$lib/components/ListItem.svelte';
 
-  const { entry, isSelected = false, onclick, skeleton = false, badge = undefined, previewDictionary = false }: {
+  interface Props extends ListItemProps {
```

Comment:
EntryRow now says that you can pass `children` but it's ignored. See comment on `SenseRow`